R/summarize_draws.R

```diff
@@ -0,0 +1,108 @@
+#' @title Calculate summary statistics
+#'
+#' @description Calculate summary statistics when collapsing over a certain
+#'   variable. For example can calculate summary statistics across a set of
+#'   draws, or locations, or location-years, etc.
+#'
+#' @param dt \[`data.table()`\]\cr
+#'   Data to calculate summary statistics for.
+#' @param id_cols \[`character()`\]\cr
+#'   ID columns that uniquely identify each row of `dt`.
+#' @param summarize_cols \[`character()`\]\cr
+#'   The `id_cols` that should be collapsed and to calculate summary statistics
+#'   over.
+#' @param value_col \[`character(1)`\]\cr
+#'   Value column that summary statistics should be calculated for.
+#' @param summary_fun \[`character()`\]\cr
+#'   Names of the functions that can be used to summarize a vector of values.
+#'   Default is "mean".
+#' @param probs \[`numeric()`\]\cr
```

Comment:
Can this be null (if, say, you just want the mean)?

Reply:
Yup added a bit more documentation on this.